### DISCUSSION TITLE
[已解决]一个页面不可以绘制两个（Ant Design Chart的）IndentedTree

### DISCUSSION THREAD

**Discussion by JackZimmer** [antvis/G2Plot#2276] [Q&A] [answered]
我的一个项目刚刚启动，看了下Ant Design Chart感觉封装的干净利落，且样式美观，准备使用Ant Design Chart当做默认图表库。

遇到了一个使用上的问题。在使用IndentedTree的时候，发现，**一个页面上不能绘制两个或者多个IndentedTree。**

复现地址：https://codesandbox.io/s/reverent-hawking-24jg4?file=/Container.tsx
![image](https://user-images.githubusercontent.com/2899765/105955893-f3c23100-60b1-11eb-953f-f85b3b00c30f.png)

我试了其他的图表，都没有问题，只有IndentedTree是这样的

**Comment by JackZimmer** [comment]
使用的Ant Design Chart版本是 "@ant-design/charts": "^1.0.16"

**Comment by visiky** [comment]
😂 这个好像是基于 G6 绘制的？ @lxfu1

**Reply by lxfu1** [reply]
> 😂 这个好像是基于 G6 绘制的？ @lxfu1

已经修复了。

**Comment by baizn** [comment, marked as answer]
修复了，详见这个pr： https://github.com/ant-design/ant-design-charts/pull/486
